<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tweezers</class>
 <widget class="QMainWindow" name="Tweezers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tweezers</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0">
    <item row="2" column="2">
     <layout class="QGridLayout" name="main_2" columnstretch="0,0,0,0">
      <item row="3" column="0" rowspan="2">
       <widget class="QListWidget" name="directoryList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="2" colspan="2">
       <layout class="QVBoxLayout" name="Main">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="globSelect">
            <property name="minimumSize">
             <size>
              <width>207</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="expSelect"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="fileList">
          <property name="editTriggers">
           <set>QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="expField"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="doRename">
            <property name="text">
             <string>Rinomina</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="undoRename">
            <property name="text">
             <string>Undo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="previewUpdate">
            <property name="text">
             <string>Anteprima</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="expList"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="selectDir"/>
        </item>
        <item>
         <widget class="QToolButton" name="selDirButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="3">
       <layout class="QVBoxLayout" name="infoLayout">
        <item>
         <widget class="QPushButton" name="hideFileInfo">
          <property name="text">
           <string>Nascondi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="fileInfo">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
